docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,428 @@
+          --cloud CLOUD_AND_REGION                \
+          --plan business-4                       \
+          --project PROJECT_NAME
+
+#. Run the following command to create an Aiven for Apache Flink service named ``demo-flink``:
+
+   ::
+
+      avn service create demo-flink               \
+          --service-type flink                    \
+          --cloud CLOUD_AND_REGION                \
+          --plan business-4                       \
+          --project PROJECT_NAME
+
+#. Add the ``demo-kafka`` and ``demo-postgresql`` integrations to the ``demo-flink`` service.
+
+   a. Enter the following command to add the ``demo-kafka`` service integration:
+
+      ::
+
+         avn service integration-create           \
+             --project PROJECT_NAME               \
+             --service-type flink                 \
+             -s demo-kafka                        \
+             -d demo-flink
+
+   b. Enter the following command to add the ``demo-postgresql`` service integration:
+
+      ::
+
+         avn service integration-create           \
+             --project PROJECT_NAME               \
+             --service-type flink                 \
+             -s demo-postgresql                   \
+             -d demo-flink
+
+   c. Enter the following command to list the integrations:
+
+      ::
+
+         avn service integration-list demo-flink
+
+      The output should show you that both ``demo-kafka`` and ``demo-postgresql`` integrations are enabled as well as the corresponding ``integration_id`` values that you need later when creating data tables.
+
+
+
+Set up sample data
+------------------
+
+Before you start, clone the `Dockerized fake data producer for Aiven for Apache Kafka <https://github.com/aiven/fake-data-producer-for-apache-kafka-docker>`_ Git repository to your computer.
+
+1. Follow `these instructions <https://developer.aiven.io/docs/tools/cli/user/user-access-token.html#manage-access-tokens>`_ to create an authentication token for your Aiven account.
+
+#. Go to the data producer tool directory and copy the ``conf/env.conf.sample`` file to ``conf/env.conf``.
+
+#. Edit the ``conf/env.conf`` file and update the parameters:
+
+   ::
+
+      PROJECT_NAME="AIVEN_PROJECT_NAME"
+      SERVICE_NAME="demo-kafka"
+      TOPIC="cpu_load_stats_real"
+      PARTITIONS=2
+      REPLICATION=2
+      NR_MESSAGES=200
+      MAX_TIME=0
+      SUBJECT="metrics"
+      USERNAME="AIVEN_ACCOUNT_EMAIL"
+      TOKEN="AUTHENTICATION_TOKEN"
+
+   Replace ``AIVEN_PROJECT_NAME`` and ``AIVEN_ACCOUNT_EMAIL`` with the details for your Aiven account, and replace ``AUTHENTICATION_TOKEN`` with the token that you created.
+
+   .. note::
+      The ``NR_MESSAGES`` option defines the number of messages that the tool creates when you run it. Setting this parameter to ``0`` creates a continuous flow of messages that never stops.
+
+#. Run the following command to build the Docker image:
```

Comment:
I think we should have explained that docker and the CLI are pre-requisites at the start of the tutorial (and that the docker bit is only to give you some sample data, you can manage without it)

Reply:
Added a "Requirements" section to both this and the console version of the tutorial.